cmd/observer/node_builder/observer_builder.go

```diff
@@ -156,7 +157,7 @@ type ObserverServiceBuilder struct {
 
 	// components
 	LibP2PNode              *p2p.Node
-	FollowerState           protocol.MutableState
+	FollowerState           stateprotocol.MutableState
```

Comment:
you didn't have to change the naming here?

Reply:
I did since I imported the package with `protocol.API` https://github.com/onflow/flow-go/pull/2962/files/5b08f9c1f0f51eafed149a86e572578868d29b49#diff-8e2393e4d4435f2229d72e4d978fcc6b45e44cc54e978011f3375709e0a143fcR37